ruby/2.3/Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
   DEBIAN_FRONTEND=noninteractive apt-get purge -y --auto-remove software-properties-common && \
   rm -rf /var/lib/apt/lists/*
 
+RUN gem update --system
```

Comment:
let's do `gem update --system 3.0.2`, since anything still using this will be old.

Reply:
updated